js/script.js

```diff
@@ -25,3 +27,31 @@ toggleMenuButton.forEach((item) => {
 seeCloseProjectButton.forEach((item) => {
   item.addEventListener('click', togglePopUp);
 });
+
+// Form validation
+
+const validationWithMessage = () => {
+  let message = '';
+  const email = contactForm.elements.email.value;
+  const styles = getComputedStyle(formErrorDiv);
+  const errorP = document.getElementById('error-message');
+  formErrorDiv.style.width = '0px';
+  formErrorDiv.style.visibility = 'hidden';
+  if (styles.visibility === 'hidden' && email !== email.toLowerCase()) {
+    formErrorDiv.style.visibility = 'visible';
+    formErrorDiv.style.width = '301.865px';
+    message = `Email Has to be in lower case, You passed: ${email}`;
+  }
+
+  errorP.textContent = message;
+  return message;
+};
+
+contactForm.addEventListener('submit', (ev) => {
+  ev.preventDefault();
+  const error = validationWithMessage();
+  if (!error) {
+    formErrorDiv.style.visibility = 'hidden';
+    contactForm.submit();
+  }
+});
```

Comment:
- [ ] hey, your form is still submitted in capital letter. Please, fix this!

Here: :point_down: 

https://user-images.githubusercontent.com/59397159/168066092-9fbab86e-9ac4-4721-853c-48cf3f9959f5.mp4

**see another** :point_down: 




https://user-images.githubusercontent.com/59397159/168067700-e6a0cdd7-8188-4534-a30e-d3dd06795f75.mp4